05_API_testing/Vesta_Moon_Mars_Trek_WMTS/Moon/GET-Apollo_15_Metric_Cam_DEM_ColorHillshade.js

```diff
@@ -0,0 +1,183 @@
+// Test 1: Validate status code (should be 200)
+pm.test("Test 1: Verify that the status code is 200", function () {
+    pm.response.to.have.status(200);
+});
+// Test 2: Validate the response time (should be less than 500)
+pm.test("Test 2: Validate the response time is less than 500ms", function () {
+    pm.expect(pm.response.responseTime).to.be.below(500);
+});
+// Test 3: Validate the response size (should be less than 2KB)
+pm.test("Test 3: Validate the response size is less than 2KB", function () {
```

Comment:
2KB != 2000B @sergei-orloff :)

Reply:
🤣 @SergioUS    I just tried to save 48 bites!